src/main.py

```diff
@@ -112,5 +112,13 @@ async def send_embed(channel: discord.TextChannel, embed: discord.Embed, message
     leah.posted.add(message.id)
     await channel.send(embed=embed)
 
+async def get_original_author(showcase_message: discord.Message):
```

Comment:
Need to indicate return type (Optional[discord.Member])

Reply:
Done! :)